content/en/docs/ibm/using-icr.md

```diff
@@ -73,7 +73,7 @@ kubectl patch serviceaccount default-editor \
 -n <my_namespace>
 ```
 
-Once the service account is patched, when you create the Notebook Server through Kubeflow dashboard, you should be able to choose a `Custom Image` then fill in a notebook image path from the ICR like this:
+The service account should be updated. Then, when you create the Notebook Server through Kubeflow dashboard, you should be able to choose a Custom Image. Then, set the notebook image path from the ICR as follows:
```

Comment:
To avoid repeating "then" (as in "Then, when.... Then, set....") maybe change the last one to "Afterwards" or "Finally" (if it's the last step). For example:

> The service account should be updated. Then, when you create the Notebook Server through Kubeflow dashboard, you should be able to choose a Custom Image. Afterwards, set the notebook image path from the ICR as follows: